src/sql/workbench/browser/modal/dialogHelper.ts

```diff
@@ -22,6 +21,11 @@ export function appendRow(container: HTMLElement, label: string, labelClass: str
 	}
 	const labelContainer = append(append(rowContainer, $(`td.${labelClass}`)), $('div.dialog-label-container'));
 	labelContainer.style.display = 'flex';
+
+	if (title) {
+		labelContainer.title = getWrappedString(title, 100);
```

Comment:
Why not just put the newlines in the description text itself? We shouldn't assume everyone is going to want to limit their text like this (especially in a very generic function like this) and that will save all this parsing and checking to find where to insert the newlines.

Reply:
Really? Well, I don't agree with that.

Text wrapping shouldn't be done in content, it takes away the extensibility of this design to other custom options. We should have this wrapping applying during rendering to every future custom option that is added to main dialog page with the 'showOnConnectionDialog' property. Wrapping is always done during rendering, never a content problem. Since this is tooltip, we cannot do it with CSS directly, so we use this dynamic insertion based on desired max width.

Reply:
At the very least you shouldn't be doing this for everything that uses that unless you go through a design review and validate that we're ok with this being done everywhere. If you make this an optional feature that callers to the function can set then that's much more reasonable to me.

Did you look how other tooltips are handled already? I can't imagine this is the first time this has come up for VS Code so there might already be a built-in solution for this instead of making our own.

Reply:
I can make it configurable.

I couldn't find any existing implementation of this, I did look through thoroughly, but if you find something that I missed please let me know. This is a special need for our case as our tooltip text is too long, they are generally less than 3-4 words in length that don't require wrapping.